api/enums.py

```diff
@@ -53,6 +53,11 @@ class EdgeTtsVoiceGender(Enum):
     FEMALE = "Female"
 
 
+class XVASynthModel(StrEnum):
+    SYNTHESIZE_URL = "http://127.0.0.1:8008/synthesize"
```

Comment:
These endpoints should be moved to the config in a new xva_synth config block (like edge_tts etc.)

Reply:
Yep, done.